Form-Controls/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  width: 100%;
+  font-size: 20px;
+  color: black;
+  margin-right: auto;
+  margin-left: auto;
+  margin-bottom: 0;
+  background: rgb(245, 206, 218);
+  padding: 0;
+}
+
+
+h1{
+  
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+input{
+  width: 400px;
+  height: 15px;
+  font-size: 20px;
+  padding: 10px;
+  border: 0px solid white;
+  border-radius: 8px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+}
+select{
+  width: 400px;
```

Comment:
Similar to the input fields, avoid using `px` for width and `margin-left: 85px` to align the element. This hard-coded margin might break on different screen sizes.
You can set the width as 100% with max-width for responsiveness.